styles/Footer.module.css

```diff
@@ -0,0 +1,215 @@
+.footer {
+    width: 100%;
+    height: 160px;
+    border-top: 1px solid #eaeaea;
+    /* overflow-y: hidden; */
+  }
+
+.topLine {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 25px 50px;
+    flex-wrap: wrap;
+}
+
+.logoBox {
+    display: flex;
+    height: 70px;
+    padding-top: 5px;
+}
+
+.logo {
+    display: inline-block;
+    height: 45px;
+    width: auto;
+    cursor: pointer;
+}
+
+.leftBox {
+    display: flex;
+    flex-grow: 1;
+    height: 70px;
+    padding-top: 5px;
+    padding-left: 25px;
+}
+
+.mapIcon {
+    height: 40px;
+}
+
+.addressBox {
+    font-size: 15px;
+    color: #18A0FB;
+}
+
+.addressBox p {
+    margin-top: .50px;
+    line-height: 10px;
+    cursor: pointer;
+}
+
+.hoursBox {
+    padding-left: 25px;
+    font-size: 15px;
+}
+
+.hoursBox p {
+    margin-top: .50px;
+    line-height: 10px;
+}
+
+.hoursTime {
+    padding-left: 25px;
+    font-size: 15px;
+}
+
+.hoursTime p {
+    margin-top: .50px;
+    line-height: 10px;
+}
+
+.rightBox{
+    display: flex;
+    height: 70px;
+}
+
+.navBox a {
+    padding-left: 50px;
+    font-weight: 500;
+}
+
+.bottomLine {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0px 50px;
+    height: 75px;
+    border-width: 50%;
+}
+
+.bottomLeftBox {
+    display:flex;
+}
+
+.bottomLeftBox p {
+    font-weight: 300;
+}
+
+.bottomRightBox {
+    display: flex;
+    align-items: center;
+    font-weight: 500;
+    color: #18A0FB;
+}
+
+.phone {
+    height: 25px;
+    padding-right: 10px;
+    margin-top: 7px;
+    cursor: pointer;
+}
+
+.envelope {
+    height: 25px;
+    padding-left: 25px;
+    padding-right: 10px;
+    margin-top: 7px;
+    cursor: pointer;
+}
+
+.fbIcon {
+    height: 25px;
+    padding-left: 30px;
+    cursor: pointer;
+    margin-top: 7px;
+}
+
+.solid {
+    border-top: 1px solid black;
+    width: 94%;
+  }
+
+@media (max-width: 1140px){
+
+    .rightBox{
+        display:none;
+    }
+
+    .topLine {
+        justify-content: center;
+        padding-bottom: 10px;
+    }
+    .logoBox {
+        justify-content: center;
+    }
+
+    .leftBox {
+        flex-grow: 0;
+        justify-content: center;
+    }
+
+    .bottomLine {
+        justify-content: center;
+        flex-direction: column;
+    }
+
+    .bottomLeftBox{
+        order:2;
+    }
+
+    .bottomRightBox {
+        padding-top: 15px;
+        order:1;
+    }
+    
+  }
+ 
+@media (max-width: 525px) {
```

Comment:
@LRode I could not solve this for the life of me. So for the mobile size (media size of 330px).

Starting at the size of 525px, I got rid of the email and phone and just kept the icons (which I thought looked nice on the mobile screen).  However, I was having an issue with the wrap of `.leftBox`. I wanted the two `.hours` boxes on the same line and staying centered while shrinking. However I couldn't get flex right :,(

Reply:
I will fix this up in my commit, but we can have a call soon if you want me to explain what exactly I did. Basically, a common pattern with flex boxes, is wrapping sections in "container' divs. So if you always want the hours to move together as a unit, wrap them in their own div, now you have a nested flex inside another flex. but now that parent flex can control the hours all together, instead of having to shuffle around both its children, and making sure they always wrap together. You'll see I did this a bunch in my commit, since it makes it easier to wrap the children properly when we get to mobile.

Reply:
Okay thank you! That makes a lot of sense to combine the two into one common div.